src/mapd-crossfilter.js

```diff
@@ -82,6 +82,22 @@ function notEmpty(item) {
   }
 }
 
+/**
+ * creates WKT POLYGON string from given points array
+ * @param points, ex: [[-180,-90], [180.-90], [180,90], [-180,90]]
+ * @returns {string}
+ */
+export function createWKTPolygonFromPoints(points) {
+  let wkt_str = "POLYGON(("
+  points.forEach((p, i) => {
+    if (i < points.length) {
+      wkt_str = `${wkt_str} ${p[0]} ${p[1]}, `
```

Comment:
Note: another syntax could be wkt_str += ` ${p[0]} ${p[1]}, `

Reply:
eliminate one duplicate code, thank you